src/router_core/route_control.h

```diff
@@ -43,6 +43,13 @@ qdr_auto_link_t *qdr_route_add_auto_link_CT(qdr_core_t          *core,
                                             qd_parsed_field_t      *connection_field,
                                             qd_parsed_field_t   *external_addr);
 
+/**
+ * Attempts re-establishing auto links across the related connections/containers
+ * @param core Pointer to the core object returned by qd_core()
+ * @param context - the auto link that needs to be activated
+ */
+void qdr_route_attempt_auto_link_CT(qdr_core_t      *core, void   *context);
```

Comment:
This is not part of the route_control API.  It is simply an internal helper within the module.  It should be removed from the header file (API) and made static in the source file.